### ISSUE TITLE
Regular testing of SPIR-V using source from OpenCL CTS

### ISSUE THREAD

**Issue by StuartDBrady** [KhronosGroup/SPIRV-LLVM-Translator#547]
We should perform regular testing of the translation to SPIR-V of IR compiled from OpenCL C source produced by the OpenCL CTS. We should use spirv-val from the SPIR-V Tools to validate the result. We should also testing translation of the SPIR-V back to LLVM IR, and unsure that the IR is valid.

**Comment by jenatali** [comment]
Agreed. The basic\test_async_copy tests when run through clang with default optimizations/settings produce LLVM containing non-constant-size memset, which fails to translate.
